tests/integration/test_resourcegroups.py

```diff
@@ -0,0 +1,109 @@
+import unittest
+import json
+from localstack.utils.aws import aws_stack
+from localstack.utils.common import short_uid
+
+
+class TestRESROUCEGROUPIntegrations(unittest.TestCase):
```

Comment:
nit: could be renamed to `TestResourceGroups`

Reply:
Done